src/main/java/net/minecraftforge/common/DimensionManager.java

```diff
@@ -136,7 +135,7 @@ public static void registerDimension(int id, DimensionType type)
         dimensions.put(id, new Dimension(type));
         if (id >= 0)
```

Comment:
I think we can get rid of this check now, no real reason to *not* save negative IDs.